tensor/stats/metric/vec.go

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) 2024, Cogent Core. All rights reserved.
```

Comment:
Call this file `vector.go`.

Reply:
stands for vectorize, not vector.  not sure I want such a longe filename.